policy-controller/k8s/index/src/default_allow.rs

```diff
@@ -0,0 +1,196 @@
+use crate::ServerRx;
+use anyhow::{anyhow, Error, Result};
+use linkerd_policy_controller_core::{
+    ClientAuthentication, ClientAuthorization, IdentityMatch, InboundServer, IpNet, NetworkMatch,
+    ProxyProtocol,
+};
+use linkerd_policy_controller_k8s_api as k8s;
+use tokio::{sync::watch, time};
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum DefaultAllow {
```

Comment:
the proxy now also has a `tls-unauthenticated` allow policy now (added in linkerd/linkerd2-proxy#1184) that allows TLS connections with or without client auth but denies all non-TLS connections. do we want to be able to set that policy from the policy controller, as well, or do we expect it to only be set by the proxy-injector?

Reply:
For now I don't think it's important to support here. We can add it later if we need it.